pipelines/LATISS/ApPipe.yaml

```diff
@@ -5,13 +5,3 @@ description: |
 instrument: lsst.obs.lsst.Latiss
 imports:
   - location: $AP_PIPE_DIR/pipelines/_ingredients/ApPipe.yaml
-    exclude:
-      # TODO: prompt_prototype does not yet support R/B analysis: DM-42220.
-      - rbClassify
-
-tasks:
-  transformDiaSrcCat:
-    class: lsst.ap.association.TransformDiaSourceCatalogTask
-    config:
-      # TODO: prompt_prototype does not yet support R/B analysis: DM-42220.
-      doIncludeReliability: False
```

Comment:
Should `pipelines/LSSTComCamSim/ApPipe.yaml` be updated too, assuming the model can be added to `/repo/ops-rehearsal-3-prep`?

Reply:
In theory, yes. But everyone seemed skeptical that that was a good idea. I can add it now, or save it for a later issue after we've seen how it works on LATISS-prod...

Reply:
Leaving it for later is totally fine